docs/citations.rst

```diff
@@ -19,11 +19,22 @@ Use in Publications
 
 Updating list of citations and use cases of :code:`pyhf`:
 
+Use Citations
+~~~~~~~~~~~~~
+
 .. bibliography:: bib/use_citations.bib
    :list: bullet
    :all:
    :style: plain
 
+General Citations
```

Comment:
Happy to rename this to something better than "general" if people have suggestions.